src/backend/auth/server.js

```
@@ -0,0 +1,94 @@
+
+app.post('/oauth2/token', async (req, res) => {
+    // console.log("Received request body:", req.body); // Debugging log
+
+    const { code } = req.body;
+    // console.log(code);
+    if (!code) {
+        console.error("Missing parameters:", req.body);
+        return res.status(400).json({ error: "invalid_request", message: "Missing required parameters" });
+    }
+
+    const authHeader = Buffer.from(`${CLIENT_ID}:${CLIENT_SECRET}`).toString('base64');
```

ESLint

[error] 'Buffer' is not defined.

(no-undef)